frontend/src/component/project/Project/Project.tsx

```diff
@@ -94,11 +94,14 @@ const TabText = styled('span')(({ theme }) => ({
 }));
 
 const ChangeRequestsLabel = () => {
+    const simplifyProjectOverview = useUiFlag('simplifyProjectOverview');
     const projectId = useRequiredPathParam('projectId');
     const { total } = useActionableChangeRequests(projectId);
 
+    const count = simplifyProjectOverview ? 0 : (total ?? 0);
```

Comment:
Isn't `if (!simplifyProjectOverview) return null;` a better check? If the flag is off we don't add empty elements to the DOM.

Reply:
No, because if we return `null`, we won't render a label at all. This way, at least the label is the same, though we could potentially just use a different label

Reply:
Ok, I thought this was just the count. I'll go ahead and approve it then.

Reply:
But I found a nicer way to do it in ad7aa879fa6b7ac1be54c97c8f14c8be8d7d3033. We'll just render the text.